andela/ah-scorpion-frontend
# #160345391 Protect Dashboard Route

#### What does this PR do?
- Protect authorize route - Dashboard
#### Description of Task to be completed?
- Protect Dashboard Route
#### How should this be manually tested?
- Clone this repository and branch to `ch-Protect-Dashboard-16034539`
- Run your local-server `npm start -s`
- Access `localhost:3000/dashboard` you should be redirected back to login page.
- Try to login authorize user and see if you can access the route
#### What are the relevant pivotal tracker stories?
[#160345391](https://www.pivotaltracker.com/story/show/160345391)
